gammapy/irf/psf_table.py

```diff
@@ -753,3 +758,412 @@ def _get_1d_table_psf(self, energy_index, **kwargs):
             self._table_psf_cache[energy_index] = table_psf
 
         return self._table_psf_cache[energy_index]
+
+class PSF3D(object):
+    """Table PSF.
+
```

Comment:
Can you add a link to
http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/psf/psf_table/index.html
to this docstring?

Here's an example how to create a Sphinx link to another Sphinx project:
https://github.com/gammapy/gammapy/blob/ba689868c6b6ca35cd230dc508918f4653319a6f/gammapy/irf/psf_king.py#L151

In your case the label is this:
https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/blame/master/source/irfs/psf/psf_table/index.rst#L3

Putting this in your docstring should work:
```
This class implements the format described here: :ref:`gadf:psf_table`.
```
You can check by running this locally:
```
python setup.py build_sphinx
# get a coffee
open docs/_build/html/index.html
# Browse to docs for your class
```